tests/Integration/Behaviour/Features/Scenario/Category/category_management.feature

```diff
@@ -1,165 +1,752 @@
+      | name[en-US]         | not important    |
+      | name[fr-FR]         | not important    |
+      | active              | true             |
+      | parent category     | home-accessories |
+      | link rewrite[en-US] | not-important    |
+      | link rewrite[en-US] | not-important    |
+    And I add new category "category_b8" with following details:
+      | name[en-US]         | not important    |
+      | name[fr-FR]         | not important    |
+      | active              | true             |
+      | parent category     | home-accessories |
+      | link rewrite[en-US] | not-important    |
+      | link rewrite[en-US] | not-important    |
+    And I add product "product_b1" with following information:
+      | name[en-US] | bottle of beer |
+      | type        | standard       |
+    And I add product "product_b2" with following information:
+      | name[en-US] | bottle of beer2 |
+      | type        | standard        |
+    And I enable product "product_b1"
+    And I enable product "product_b2"
+    And product "product_b1" should be enabled
+    And product "product_b2" should be enabled
+    And I assign product product_b1 to following categories:
+      | categories       | [category_b6,category_b7,category_b8] |
+      | default category | category_b7                           |
+    And I assign product product_b2 to following categories:
+      | categories       | [category_b6,category_b7,category_b8] |
+      | default category | category_b6                           |
+    Then product "product_b1" should be assigned to following categories:
+      | id reference | name          | is default |
+      | category_b6  | not important | false      |
+      | category_b7  | not important | true       |
+      | category_b8  | not important | false      |
+    Then product "product_b2" should be assigned to following categories:
+      | id reference | name          | is default |
+      | category_b6  | not important | true       |
+      | category_b7  | not important | false      |
+      | category_b8  | not important | false      |
+    When I bulk delete categories "category_b6,category_b7,category_b8" choosing mode "associate_only"
+    Then category "category_b6" does not exist
+    And category "category_b7" does not exist
+    And category "category_b8" does not exist
+    Then product "product_b1" should be assigned to following categories:
+      | id reference     | name             | is default |
+      | home-accessories | Home Accessories | true       |
+    Then product "product_b2" should be assigned to following categories:
+      | id reference     | name             | is default |
+      | home-accessories | Home Accessories | true       |
+    And product "product_b1" should be enabled
+    And product "product_b2" should be enabled
+
+  Scenario: Bulk delete categories which are the last remaining associated categories for some product
+  by using associate_and_disable deletion mode
```

Comment:
```suggestion
  Scenario: Bulk delete categories which are the last remaining associated categories for some product by using associate_and_disable deletion mode
```
Scenario details should remain on the same line no? Or is it accepted by Gerkhin syntax?

Reply:
it works, so I thought why not. Easier to read :smile: